cpp/ql/test/library-tests/dataflow/taint-tests/map.cpp

```diff
@@ -321,19 +321,19 @@ void test_unordered_map()
 	sink(m12.at("abc") = "def");
 	sink(m13.at("abc") = source()); // $ ast,ir
 	sink(m10["abc"]);
-	sink(m11["abc"]); // $ ast MISSING: ir
+	sink(m11["abc"]); // $ ast,ir
 	sink(m12["abc"]);
-	sink(m13["abc"]); // $ ast MISSING: ir
+	sink(m13["abc"]); // $ ast,ir
 
 	// ranges
 	std::unordered_map<char *, char *> m14;
 	m14.insert(std::make_pair("a", "a"));
 	m14.insert(std::make_pair("b", source()));
 	m14.insert(std::make_pair("c", source()));
 	m14.insert(std::make_pair("d", "d"));
-	sink(m2.equal_range("b").first); // $ ir
-	sink(m2.equal_range("b").second); // $ ir MISSING: ast
-	sink(m2.equal_range("c").second); // $ SPURIOUS: ir
+	sink(m2.equal_range("b").first);
+	sink(m2.equal_range("b").second); // $ MISSING: ast,ir
+	sink(m2.equal_range("c").second);
```

Comment:
I wonder if these cases (and the similar cases in `test_pair`) are supposed to be on `m14` rather than `m2`?

Reply:
Indeed. It certainly looks weird that there's a bunch of insertions into `m14`, followed by a bunch of lookups on `m2`.

Reply:
I think we should fix this as a follow-up (doing it on this PR might make it harder to understand).  Would you like me to do this, or would you rather co-ordinate it yourself?

Reply:
Good idea. I can do it myself after this has been merged

Reply:
(done in https://github.com/github/codeql/pull/7102)